frontend/app.vue

```diff
@@ -1,4 +1,5 @@
 <script setup lang="ts">
+import { isSilent } from '~/lib/error'
```

Comment:
nit: `@/lib/error`

Reply:
Lol not sure how I missed that the rest of the codebase does it that way, done